test/system/060-mount.bats

```diff
@@ -173,6 +173,35 @@ load helpers
     run_podman rm -t 0 -f $cid
 }
 
+@test "podman mount containers.conf" {
+    tmpfile1=$PODMAN_TMPDIR/volume-test1
+    random1=$(random_string 30)
+    echo $random1 > $tmpfile1
+
+    tmpfile2=$PODMAN_TMPDIR/volume-test2
+    random2=$(random_string 30)
+    echo $random2 > $tmpfile2
+
+    mountStr1=type=bind,src=$tmpfile1,destination=$tmpfile1,ro
+    mountStr2=type=bind,src=$tmpfile2,destination=$tmpfile1,ro
+    containersconf=$PODMAN_TMPDIR/containers.conf
+    cat >$containersconf <<EOF
+[containers]
+mounts=[ "$mountStr1", ]
+EOF
+
+    run_podman 1 run $IMAGE cat $tmpfile1
+    is "$output" "cat: can't open '$tmpfile1': No such file or directory" "$tmpfile1 does not exist"
+
+    CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman run $IMAGE cat $tmpfile1
+    is "$output" "$random1" "file should container $random1"
+
+    CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman run --mount $mountStr2 $IMAGE cat $tmpfile1
+    is "$output" "$random2" "file should container $random2"
```

Comment:
```suggestion
    is "$output" "$random2" "file should contain $random2"
```

Reply:
It's probably worth documenting that containers.conf mounts can be overridden on the CLI.

Reply:
That should be in containers.conf not in Podman, but that is true of all fields.